grails-app/controllers/org/pih/warehouse/reporting/IndicatorApiCommand.groovy

```diff
@@ -1,10 +1,12 @@
 package org.pih.warehouse.reporting
 
 import grails.validation.Validateable
+import java.time.Instant
+
 import org.pih.warehouse.core.Location
 
 class IndicatorApiCommand implements Validateable {
     Location facility
-    Date startDate
-    Date endDate
+    Instant startDate
+    Instant endDate
```

Comment:
Used in:
- reportService.getProductsInventoried to fetch from Transaction.transactionDate (Date)
- reportService.getInventoryAccuracy to fetch from CycleCountProductSummary.transactionDate (Instant)
- reportService.getInventoryShrinkage to fetch from InventoryAuditDetails.transactionDate (Instant)

So even though we pick a date-only, these are really full date-times.

Note that because reportService.getProductsInventoried queries Transaction, we need to convert these Instants to Dates (see NumberDataService.getItemsInventoriedInRange). Once we refactor Transaction to use Instant, that conversion can be removed